File: 0657-robot-return-to-origin/0657-robot-return-to-origin.py
class Solution:
    def judgeCircle(self, moves: str) -> bool:
        h = 0
        v = 0
        for i in moves:
            if i == 'U':
                h += 1
            elif i == 'D':
                h -= 1
            elif i == "R":
                v += 1
            else:
                v -= 1
        return h == 0 and v == 0

File: 0657-robot-return-to-origin/0657-robot-return-to-origin.c
bool judgeCircle(char* moves) {
    int v = 0;
    int h = 0;
    for(char* i = moves; *i != '\0'; i++){
        if (*i == 'U') v++;
        else if (*i =='D') v--;
        else if (*i == 'R') h++;
        else h --;
    }
    return v == 0 && h == 0;
}